gcloud/pubsub/test_subscription.py

```diff
@@ -620,29 +603,68 @@ def test_check_iam_permissions_w_alternate_client(self):
 
         self.assertEqual(len(allowed), 0)
         self.assertEqual(len(conn1._requested), 0)
-        self.assertEqual(len(conn2._requested), 1)
-        req = conn2._requested[0]
-        self.assertEqual(req['method'], 'POST')
-        self.assertEqual(req['path'], PATH)
-        self.assertEqual(req['data'], REQUESTED)
+        self.assertEqual(len(conn2._requested), 0)
+        self.assertEqual(conn2._tested_iam_permissions,
+                         (self.SUB_PATH, ROLES))
 
 
-class _Connection(object):
+class _Connection(object):  # pylint: disable=too-many-instance-attributes
```

Comment:
> Ick on the disable. What is the limit and how far does this code exceed it.

Don't know, don't care:  I see no value in rearchitecting this class to satisfy it.

> Can you just add an over-ride?

The disable is block-scoped:  it affects only the current class.